### ISSUE TITLE
Errors and failure to write correctly

### ISSUE THREAD

**Issue by mkrupczak3** [arcadenea/iso2opl#4]
Hello,

I'm using a PS2 with FreeMCBoot and OPL v1.0.0

Have tried writing a few iso's to USB, but sometimes OPL doesn't recognize the files, and sometimes when it DOES recognize the files, the game starts to boot but then quickly fails. 

Is this due to bugs with iso2opl or am I missing necessary launch conditions that are game-specific?

**Comment by ghost** [comment]
@mkrupczak3 
Hello how are you? 
Your problem may be related to file fragmentation...
I use linux and I have the custom to always perform the conversion first to a folder and then pass the files to the USB stick, thus avoids fragmentation of the files, because the USB stick has the slowest throughput. 

I think it's a good tip for you to :)

**Comment by mkrupczak3** [comment]
Think for me it was fragmenting because disk was too full

My mistake for sure, but FWIW iso2opl didn't inform me of this error before exiting

Good advice re: fragmenting. Initiating the conversion in a directory on my Computer's main filesystem would have circumvented this error

I'm using a HDD now instead of OPL, so this doesn't effect me much anymore. Sharing in case it's helpful for other software preservationists booting off of USB sticks

**Comment by ghost** [comment]
Yes, thanks for all, i create a pull request do put this tip in the README.md.

Att, Maicon Douglas Stelzer.
